client/src/components/UIElements/DoubleOptionToggleSwitch/DoubleOptionToggleSwitch.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react';
+
+const DoubleOptionToggleSwitch = ({ labelFirst, labelSecond, taskFunktion }) => {
+    //true is option 1, false option 2
+    const [switchState, setSwitchState] = useState(true);
+    const toggleHandler = () => {
+        setSwitchState(!switchState);
+        taskFunktion();
```

Comment:
There are some typos in taskFunktion -> taskFunction. There are just in names but I think it is worth to change.

Reply:
Definitely You're right. I will fix this